trustmanager/x509filestore.go

```diff
@@ -213,10 +240,10 @@ func (s X509FileStore) GetVerifyOptions(dnsName string) (x509.VerifyOptions, err
 }
 
 func fileName(cert *x509.Certificate) (string, CertID, error) {
-	keyID, err := fingerprintCert(cert)
+	certID, err := fingerprintCert(cert)
```

Comment:
method is called `fileName`.

It returns a path and a certificate ID.

Recommend `getRelPathAndCertIDFromx509` or some shiz